src/components/main/App.tsx

```diff
@@ -1,3 +1,6 @@
+/*global chrome*/
+/*src/components/main/App.tsx*/
```

Comment:
What does this comment mean? I know the global tell typescript that this is a global variable and needs not to worry about the declaration. But what the /*src/components/main/App.tsx*/ means?

Reply:
So I was looking some documentation for how content scripts work, and I saw that there was a comment at the top which specified the path of the file. Originally, I thought that this was necessary for content scripts but I saw that it didn't make any difference. I don't believe it does anything, but I can remove the comment in my code and make a new pull request.

Reply:
sure, remove only the second line and push your changes. You do not need to make a new pull request.